app/templates/teachers/teacher-student-view.jade

```diff
@@ -164,65 +164,78 @@ block content
-                if !course
-                  - continue;
-                - var instance = view.courseInstances.findWhere({ courseID: course.id, classroomID: view.classroom.id })
-                - if (instance && instance.hasMember(view.user))
-                  if course
-                    .course-row.alternating-background
-                      .course-info= course.getTranslatedName()
-                      - var levels = view.classroom.getLevels({courseID: course.id}).models
-                      each level, index in levels
-                        - var levelNumber = view.classroom.getLevelNumber(level.get('original'), index + 1)
-                        - var levelProgress = view.levelProgressMap[level.get('original')]
-                        +studentLevelProgressDot(levelProgress, level, levelNumber, course)
+              - var course = _.find(view.courses.models, function(c) {return c.id === versionedCourse._id;});
+              if !course
+                - continue;
+              - var instance = view.courseInstances.findWhere({ courseID: course.id, classroomID: view.classroom.id })
+              - if (instance && instance.hasMember(view.user))
+                .course-row.alternating-background
+                  .course-info= course.getTranslatedName()
+                  - var levels = view.classroom.getLevels({courseID: course.id}).models
+                  each level, index in levels
+                    - var levelNumber = view.classroom.getLevelNumber(level.get('original'), index + 1);
+                    - var levelProgress = view.levelProgressMap[level.get('original')];
+                    +studentLevelProgressDot(levelProgress, level, levelNumber, course)
 
-      
-                      
-        //- if status == 'enrolled'
-        //-   div
-        //-     h4
-        //-       span= view.user.get('name')
-        //-       span.spr
-        //-       span(data-i18n="teacher.not_assigned")
-        //-       span :
-        //-     each versionedCourse in view.classroom.get('courses') || []
-        //-       - var course = _.find(view.courses.models, function(c) {return c.id === versionedCourse._id;});
-        //-       if !course
-        //-         - continue;
-        //-       - var instance = view.courseInstances.findWhere({ courseID: course.id, classroomID: view.classroom.id })
-        //-       - if (!(instance && instance.hasMember(view.user)))
-        //-           div.small-details
-        //-             span=course.getTranslatedName()
-                
-        //- else
-        //-   p Apply a license to this student to assign additional courses!
-        //-   button.enroll-student-button.btn.btn-navy(data-i18n="teacher.apply_license", data-user-id=view.user.id, data-event-action="Teachers Class Enrollment Enroll Student")
-        //-   // this button doesn't work yet
-        
+        .student-levels
```

Comment:
Should this class be different? Or renamed to be more generic, seems like it refers to the above div, but this new div is just using the same styling.

Reply:
Yea, it sure is.  Will rename.